packages/langchain/src/orchestration/util.ts

```diff
@@ -123,6 +126,23 @@ function mapAzureOpenAiToLangchainToolCall(
   }
 }
 
+/**
+ * Maps {@link OrchestrationToolCallChunk} to LangChain's {@link ToolCallChunk}.
+ * @param toolCallChunks - The {@link OrchestrationToolCallChunk} in a stream response chunk.
+ * @returns An array of LangChain {@link ToolCallChunk}.
+ */
+function mapOrchestrationToLangchainToolCallChunk(
+  toolCallChunks: OrchestrationToolCallChunk[]
+): ToolCallChunk[] {
+  return toolCallChunks.map(chunk => ({
+    name: chunk.function?.name,
+    args: chunk.function?.arguments,
+    id: chunk.id,
+    index: chunk.index,
+    type: 'tool_call_chunk'
+  }));
+}
+
 /**
  * Maps the completion response to a {@link ChatResult}.
  * @param completionResponse - The completion response to map.
```

Comment:
See this function, we actually do `choices.map()`

Reply:
This is still valid and necessary as orchestration service supports multiple choices for normal chat completion. 
It is only for streaming that it doesn't support multiple choices.